autoload/vimlparser.vim

```diff
@@ -3914,51 +3919,51 @@ function! s:Compiler.compile(node)
   if a:node.type == s:NODE_TOPLEVEL
     return self.compile_toplevel(a:node)
   elseif a:node.type == s:NODE_COMMENT
-    return self.compile_comment(a:node)
+    call self.compile_comment(a:node)
```

Comment:
Yes. I understand the problem, but i said this function is not performance sensitive functions compared with parser functions.

And I thought there may be readability problem with lots of "return s:NIL".